src/modules/mongodb/mongodb-container.test.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment node
```

Comment:
I believe this issue is because mongoose 6 does not support node 10, which testcontainers currently supports. Could you try to see if mongoose 5 works? Else I think we should enforce a minimum node version of 12

Reply:
Worked under mongoose 5.13.15 on node version 10.24.1 locally, so I guess we can go with that mongoose version. Don't know why but couldn't get tests starting under node version 10.0.0.